go.graphics.swing/src/main/java/go/graphics/swing/text/JOGLTextDrawer.java

```diff
@@ -20,31 +20,43 @@
 import go.graphics.text.TextDrawer;
 
 import java.awt.Font;
+import java.awt.GraphicsDevice;
+import java.awt.GraphicsEnvironment;
 import java.awt.geom.Rectangle2D;
 
 /**
  * This class is a text drawer used to wrap the text renderer.
- * 
+ *
  * @author michael
  */
 public final class JOGLTextDrawer implements TextDrawer {
 
-	private static final String FONTNAME = "Arial";
+	private static final String FONT_NAME = "Arial";
+
+	private static final int DEFAULT_DPI_WIDTH = 1920;
+	private static final float SCALING_FACTOR;
+
+	static {
+		GraphicsDevice graphicsDevice = GraphicsEnvironment.getLocalGraphicsEnvironment().getDefaultScreenDevice();
+		int screenWidth = graphicsDevice.getDisplayMode().getWidth();
```

Comment:
From what I found on the internet, the best idea is to regularily check / recalculate the scale. Another thing: @PayDay1: I just found `Toolkit.getDefaultToolkit().getScreenResolution();` which returns a dpi number. I think using this would be better than using the screen resolution (a screen can have 4k but be 50 inch large)...